developers/products/interchain-messaging-agent/overview.adoc

```diff
@@ -0,0 +1,17 @@
+=== Interchain Messaging Agent
+
+SKALE makes it possible to manage tokens for end-users between Ethereum and your SKALE Chain. You can use the Interchain Messaging Agent (IMA) to transfer ETH, ERC20, ERC721, or Dai to a deposit box on Ethereum, and SKALE will create clones of the token on your SKALE Chain. You can then send the tokens back to your end-users, and the tokens will be released from the deposit box back to the end-user. 
```

Comment:
Looks like we need to update this classic IMA description:
ERC1155, any stable coin, any NFT
Or if you want to use custom tokens see <How to use arbitrary messages in IMA>